test.c

```diff
@@ -0,0 +1,17 @@
+#include <sys/syscall.h>
+#include <linux/futex.h>
+#include <stdatomic.h>
+#include <stddef.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <time.h>
+
+
+void main()
+{
+    volatile _Atomic int a = 1;

```

Comment:
>_Atomic [](start = 13, length = 7)

looks like this test.c folder was just for testing and doesn't need to be checked in